com.unity.netcode.gameobjects/Runtime/Interest/RadiusInterestKernel.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections.Generic;
+using Unity.Netcode.Interest;
+using UnityEngine;
+
+namespace Unity.Netcode
+{
+    public class RadiusInterestKernel : IInterestKernel<NetworkObject>
+    {
+        public float Radius = 0.0f;
```

Comment:
How is it expected that devs would modify or configure values on a kernel ?

Reply:
For example in tests we do:

```
var naiveRadiusNode = new InterestNodeStatic<NetworkObject>();
var naiveRadiusKernel = new RadiusInterestKernel();
naiveRadiusKernel.Radius = 1.5f;
```